README.md

```diff
@@ -1,46 +1,39 @@
+In the modern job market, finding the right job can be a daunting task, especially for students and
+new job seekers. Our project aims to simplify this process by developing an intelligent job search
+website powered by AI and machine learning, guided by a series of questions from the user.
+What sets our project apart is its dual focus on providing highly personalized job
+recommendations and leveraging advanced data analytics to understand and predict job market
+trends. By combining web scraping, APIs, text mining, and machine learning, we aim to create a
+platform that not only matches users with relevant job opportunities, but also provides insights
+into the factors that influence job success.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Our platform is designed to bridge the gap between job seekers and employers by offering a
+seamless, data-driven job search experience. Through user-friendly interfaces and powerful
+backend analytics, we aspire to make job searching more efficient and effective. Whether it's
+understanding the correlation between certain skills and job availability or predicting the best
+locations for specific job roles, our project is geared towards making informed job searching a
+reality.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Tech Stack:
 
-### `npm run eject`
+This repository contains code written in TypeScript with React that integrates with a backend written in Python.
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+### Linter:
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+ESLint with Prettier for TypeScript.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### `npm start`
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+After running, please open [http://localhost:8080](http://localhost:8080) to view the application in the browser.
```

Comment:
It is running on the 3001 port now. So, can you change this or can you remove it since it is written in infra repo?

Reply:
Yes, I forgot to update this! It's been removed.